src/app/clusters/door-lock-server/door-lock-server.cpp

```diff
@@ -181,7 +184,13 @@ bool DoorLockServer::SetPrivacyModeButton(chip::EndpointId endpointId, bool isEn
     return SetAttribute(endpointId, Attributes::EnablePrivacyModeButton::Id, Attributes::EnablePrivacyModeButton::Set, isEnabled);
 }
 
-bool DoorLockServer::TrackWrongCodeEntry(chip::EndpointId endpointId)
+/**
+ * @brief Handles a wrong code entry attempt. If the number of wrong entry attempts exceeds the max limit, engage lockout.
```

Comment:
Again, these comments should be in the header, not here.

Reply:
haven't addressed this yet :) will do

Reply:
done